lib/DxilContainer/DxilContainerReader.cpp

```diff
@@ -1,93 +1,86 @@
 ///////////////////////////////////////////////////////////////////////////////
 //                                                                           //
-// DxilContainerReader.cpp                                                   //
+// DxilContainerReader.h                                                     //
 // Copyright (C) Microsoft Corporation. All rights reserved.                 //
 // This file is distributed under the University of Illinois Open Source     //
 // License. See LICENSE.TXT for details.                                     //
 //                                                                           //
-// Provides support for manipulating DXIL container structures.              //
+// Helper class for reading from dxil container.                                  //
 //                                                                           //
 ///////////////////////////////////////////////////////////////////////////////
 
-#include "dxc/Support/Global.h"
-#include "dxc/Support/Unicode.h"
-#include "dxc/Support/WinIncludes.h"
-#include "dxc/DXIL/DxilSubobject.h"
-#include "dxc/DxilContainer/DxilContainerReader.h"
-#include "dxc/DxilContainer/DxilRuntimeReflection.h"
+#pragma once
+
+#include "dxc\Support\Global.h"
+#include "dxc\Support\WinAdapter.h"
+#include "dxc\DxilContainer\DxilContainer.h"
+#include "dxc\DxilContainer\DxilContainerReader.h"
```

Comment:
You'll need to switch thes backslashes to slashes (or do we call them whacks?) for it to work on Linux.